crates/solvers/src/boundary/legacy.rs

```diff
@@ -578,6 +581,22 @@ fn to_boundary_auction_result(notification: &notification::Notification) -> (i64
 
     let auction_result = match &notification.kind {
         Kind::EmptySolution => AuctionResult::Rejected(SolverRejectionReason::NoUserOrders),
+        Kind::SimulationFailed(tx) => AuctionResult::Rejected(
+            SolverRejectionReason::SimulationFailure(TransactionWithError {
+                error: "".to_string(),
+                transaction: SimulatedTransaction {
+                    from: tx.from.into(),
+                    to: tx.to.into(),
+                    data: tx.input.clone().into(),
+                    internalization: InternalizationStrategy::SkipInternalizableInteraction,
+                    block_number: Default::default(),
+                    tx_index: Default::default(),
```

Comment:
Does it make sense to have the `tx_index` here?
If we are simulating on the latest block the `tx_index` shouldn't make a difference because the simulator can't know what other transactions would come before our tx_index, right?

Reply:
Not really useful, at least we did not use it, I am not sure why it was originally added.